src/mame/drivers/a7150.cpp

```diff
@@ -136,15 +145,33 @@ WRITE_LINE_MEMBER(a7150_state::a7150_tmr2_w)
 	m_uart8251->write_txc(state);
 }
 
+WRITE_LINE_MEMBER(a7150_state::ifss_loopback_w)
+{
+	m_ifss_loopback = !state;
```

Comment:
When this flips between true and false, you also need to `m_uart8251->write_rxd`, `m_rs232->write_txd`, etc. to force the correct one to the TxD output etc. otherwise the state after enabling/disabling loopback won't be right until the next output change.